File: June-18/09-06-18/java/01-Riya-in-the-angelhack.java

import java.util.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        if(n % 3 == 0 && n % 5 == 0)
            System.out.println("AngelHack!");
        else if(n % 3 == 0)
            System.out.println("Angel");
        else if(n % 5 == 0)
            System.out.println("Hack");
        else
            System.out.println(n);
        

    }
}

File: June-18/09-06-18/python/01-Riya-in-the-angelhack.py
n = int(input())
if(n % 3 == 0 and n % 5 == 0):
    print("AngelHack!")
elif(n % 3 == 0):
    print("Angel")
elif(n % 5 == 0):
    print("Hack")
else:
    print(n)
